### ISSUE TITLE
Stuck in an endless loop

### ISSUE THREAD

**Issue by WxxShirley** [ejones/sharedb-codemirror#4]
Hi， I have used this in React Framework.
In my test file, I wrote like this:
```javascript
import React, {Component} from 'react';

import otText from 'ot-text'
import shareDB from 'sharedb/lib/client'
import shareDBCodeMirror from 'sharedb-codemirror'
import CodeMirror from 'codemirror'
require('codemirror/theme/eclipse.css');
require('codemirror/mode/clike/clike');

shareDB.types.map['json0'].registerSubtype(otText.type);


class CollaberateView extends Component
{
    constructor(props){
        super(props);
        this._refs={}
    }

    componentDidMount(){

        this.editor = CodeMirror.fromTextArea(this._refs.textArea,{mode:"text/x-c++src",lineNumbers:"true",theme:"eclipse"});

        this.shareConnection = new shareDB.Connection(new WebSocket("ws://localhost:8080/share/e83b2e2695c940008195cc6375b815cf"));

        this.doc = this.shareConnection.get('example', 'textarea');
         
        try{
            shareDBCodeMirror.attachDocToCodeMirror(this.doc, this.editor, {
                key: 'content',
                verbose: true
             });
        }catch(err){
            console.log("attach doc error:"+err);
        }
        console.log("finish")
    }

    render(){
        return (
            <div className="container">
                <textarea ref={e => this._refs.textArea=e}>  </textarea>
            </div>
        )
    }
}
export default CollaberateView;
```

However, once this page was opened and I input some words, it will stuck in an endless loop like this:
![image](https://user-images.githubusercontent.com/48085932/89602748-fded3a80-d899-11ea-8909-f3e6ca36cd31.png)

**Comment by WxxShirley** [comment]
sovled. 
maybe the problem that the mode of "shareDBCodeMirror" options are not acceptable to backend(with sharedb)

**Comment by ejones** [comment]
Hey @WxxShirley, I just merged a pull request that updates this library to work with ShareDB 1.x. Previously, it was based on an old version of ShareDB, with a slightly different API for ops. Maybe that's the source of the problem? I just published `v0.3.0` to NPM - that may fix this issue.

**Comment by WxxShirley** [comment]
Yeah, that's exactly the problem , thanks!
